tools/openshift/ocp-ipi-aws/prep_for_subm.sh

```diff
@@ -1,82 +1,84 @@
 #!/bin/sh
 set -e
+
+# Script args
+OCP_INS_DIR="$(realpath ${1:-.})"
+(( $# == 1 )) || shift && TERRAFORM_ARGS=$@ # e.g. -auto-approve -lock-timeout=3m
+
+# Script args
 OCP_INS_DIR="$(realpath ${1:-.})"
```

Comment:
This ends up setting `OCP_INS_DIR` twice, is that intentional?

Reply:
@Stephen Kitt 
right, a duplicate paste...  will fix it